add-on/_locales/en/messages.json

```diff
@@ -40,7 +40,7 @@
     "description": "A menu item in Browser Action pop-up (panel_quickImport)"
   },
   "panel_openWebui": {
-    "message": "Open Web UI",
+    "message": "View Node Status",
```

Comment:
Note we use this in two places, so this changes the labels in the browser action menu as well. 
Perhaps welcome page buttons should get their own keys in translation file?

Reply:
Ah, crud, I didn't catch that - thank you!